charts/rancher-vsphere-cpi/questions.yaml

```diff
@@ -60,3 +60,54 @@ questions:
         description: vSphere tag which will used to define availability zones
         type: string
         group: Configuration
+
+  - variable: vCenter.ipFamily
+    label: Node Address IP Family
+    description: "IP Family enables the ability to support IPv4 or IPv6. Supported values are: ipv4 - IPv4 addresses only (Default) and ipv6 - IPv6 addresses only"
```

Comment:
The fact that this takes a list makes me suspect that you can actually specify multiple families, and the order sets the preference for the cluster. Is that correct? If so, the description here is incorrect, and should be revised to make it clear that you can pick multiple options, and the order matters.

If this is not correct, and you cannot specify multiple, then how do you actually enable dual-stack clusters?

Reply:
Order doesn't matter but you can pick multiple, I'll reword.

Reply:
My experience with other CPIs is that the order does matter. If you want a dual-stack cluster with IPv6 as the primary, you need to configure it appropriately.

Reply:
I've tried it out and you're correct the order does effect which address is shown with `kubectl get nodes` and on the rancher cluster management page.

It's unclear what this affects but i've updated the description to make note of this.